index.html

```diff
@@ -1521,7 +1518,9 @@
             dstLines.push(line);
         } else if (line.startsWith("\\ No newline at end of file")) {
             continue;
-            } else {
+        } else if (line == "") {
```

Comment:
Perhaps use `===`?

Reply:
That seems to be the standard in this project, so ok, changing it. I think verifying type seems unnecessary here though. The relevance of the test is to see if line resolves to an empty string.